plover_speaker_id/lookup.py

```diff
@@ -12,17 +17,27 @@
 }
 
 
+
 class SpeakerTable:
   def __init__(self):
     # TODO: load speaker IDs from external file in Plover's config directory
```

Comment:
This _is_ what you're doing, right? Why is this comment still here?

Reply:
I forgot to remove it...